eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,10 +1,25 @@
 stages:
   - stage: Build
+    dependsOn: []
+    # Purpose must be set at the level of the pipeline to enable this scenario
+    condition: eq(variables['PipelineType'], 'unified')
     jobs:
       - template: ../jobs/archetype-sdk-client.yml
 
-  - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+    - stage: LiveTest
+      # Purpose must be set at the level of the pipeline to enable this scenario
+      condition: eq(variables['PipelineType'], 'tests')
```

Comment:
Can we add a Skip variable for the condition.

Reply:
It's possible but the only situation where it'd be used is when running the live tests pipeline if we also include the proposal above to only run live tests in the ` - tests` pipeline.